Test/Mono.Cecil.Tests/ModuleTests.cs

```diff
@@ -294,5 +294,28 @@ public void ReadAndWriteFile ()
 			using (var module = ModuleDefinition.ReadModule (path))
 				Assert.AreEqual ("Foo.Foo", module.Types [1].FullName);
 		}
+
+		[Test]
+		public void ExceptionInWriteDoesNotKeepLockOnFile ()
+		{
+			var path = Path.GetTempFileName ();
+
+			var original = ModuleDefinition.CreateModule ("FooFoo", ModuleKind.Dll);
```

Comment:
the caller of ModuleDefinition.CreateModule is responsible for disposing of it? does the test still fail with that dispose?